index.html

```diff
@@ -145,6 +145,11 @@ <h2>
         For example, the badge content '7' should be displayed as '7' in the
         <a data-cite="ltli/#locale">locale</a> 'en-NZ' but as '٧' in 'ar-EG'.
       </p>
+      <p class="note">
+        The user agent can clear the badge at any time, though this
+        should generally only be done in extreme circumstances, such as
```

Comment:
Perhaps we can just say "User agents can clear the badge if necessary, e.g. due to ...". I don't know if we should call out that user agents should avoid it, it seems like it would work against them to not show the badge at some points in time.

Also is hiding different from clearing? We hide the badge when the app is closed because on some platforms that's necessary. The badge state also may go out of date if the app is closed (since we don't have SW support. Furhter, I'm not sure if we restore the state when the app is re-launched?